servers/frontend-server/src/app/Main.tsx

```diff
@@ -22,6 +22,9 @@ import { ServerError } from './Error';
 import { PersistGate } from 'redux-persist/integration/react';
 import { persistStore, persistReducer } from 'redux-persist';
 
+import { BrowserRouter as Router } from "react-router-dom";
+import Auth0ProviderWithHistory from "../../../../packages-modules/counter/browser/src/autho/auth-provider-with-history";
```

Comment:
it should be `import { Auth0ProviderWithHistory } from '@sample-stack/counter-moudle-browser`

Reply:
Okay Sure.